libs/blocks/merch-card/merch-card.js

```diff
@@ -353,6 +362,8 @@ const init = async (el) => {
   }
   let footerRows;
   if (cardType === MINI_COMPARE_CHART) {
+    const container = el.closest('[data-status="decorated"]');
```

Comment:
Any chance this doesn't exist? If so, it will cause errors on the next line, a check should be put in place